VideoLocker/src/main/java/org/edx/mobile/util/observer/AsyncObservable.java

```diff
@@ -0,0 +1,37 @@
+package org.edx.mobile.util.observer;
+
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.Executor;
+
+public enum AsyncObservable {
```

Comment:
Can you add some comments about use cases/contracts for these different observables?